### PR TITLE
Refactor/balance activity

### PR DESCRIPTION
**Pull request by kyranjamie**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/blockstack/ux/actions/runs/1337866461).

Contains commit from another PR, will be rebased after https://github.com/blockstack/stacks-wallet-web/pull/1747 is merged cc/ @beguene

- Refactors `<BalancesAndActivity />` at a component level, breaking these up into distinct features.
- Also removes a lot of dead code found with `ts-unused-export`

Next, I'd like to review what data's being consumed from the store and see if this can be simplified in anyway. This can be done in another PR though.

### CODE REVIEW

File: src/pages/home/home.tsx [hunk 1]
```diff
@@ -1,33 +1,40 @@
...
-import { UserAccount } from '@pages/home/components/user-area';
-import { HomeActions } from '@pages/home/components/actions';
 import { HiroMessages } from '@features/hiro-messages/hiro-messages';
-
+import { ActivityList } from '@features/account-activity/account-activity';
+import { BalancesList } from '@features/balances-list/balances-list';
 import { usePromptUserToSetDiagnosticPermissions } from '@common/hooks/use-diagnostic-permission-prompt';
 
-export const PopupHome = () => {
+import { HomeTabs } from './components/home-tabs';
```
Comment (beguene) [inline src/pages/home/home.tsx:15]: maybe a non-relative path is better ?

**Review by beguene — APPROVED** [review]

File: src/features/account-activity/account-activity.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React, { useMemo } from 'react';
```
Comment (fbwoolf) [inline src/features/account-activity/account-activity.tsx]: I am trying to do this too in the files I touch. Separate imports of packages vs what is from repo ...also trying to alphabetize best I can. 👍

File: src/features/account-activity/account-activity.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+import { useAccountActivity } from '@store/accounts/account.hooks';
+import { useCurrentAccountLocalTxids } from '@store/accounts/account-activity.hooks';
+import { TransactionList } from '@components/popup/transaction-list';
+import { createTxDateFormatList } from '@common/group-txs-by-date';
+import { LocalTxList } from '@features/local-transaction-activity/local-tx-list';
+
+import { NoAccountActivity } from './components/no-account-activity';
+
+export const ActivityList = () => {
```
Comment (fbwoolf) [inline src/features/account-activity/account-activity.tsx]: Should this folder/file name be `activity-list` to be consistent with `balances-list`?

File: src/routes.tsx [hunk 4]
```diff
@@ -24,7 +24,7 @@ import { Navigate } from '@components/navigate';
 import { AccountGate } from '@pages/account-gate';
 import { AccountGateRoute } from '@pages/account-gate-route';
 import { Unlock } from '@pages/unlock';
-import { PopupHome } from '@pages/home/home';
+import { Home } from '@pages/home/home';
```
Comment (fbwoolf) [inline src/routes.tsx:27]: Nice! 😄

**Review by fbwoolf — APPROVED** [review]
Fantastic! Love some of the relocation/name changes. 👏

File: src/pages/home/home.tsx (hunk 1 above)
Comment (kyranjamie) [reply src/pages/home/home.tsx:15]: Absolute paths are great when drilling down from the root, or going up and across directories.

Here its nested in the same folder, it's the shorter, clearer path imo. Maybe there's some convention we can agree on when to use absolute/relative paths. (I would suggest always absolute if going up a directory level)

File: src/features/account-activity/account-activity.tsx (hunk 3 above)
Comment (kyranjamie) [reply src/features/account-activity/account-activity.tsx]: Yes, it should. Good spot

File: src/pages/home/home.tsx (hunk 1 above)
Comment (kyranjamie) [reply src/pages/home/home.tsx:15]: Totally up for tightening rules here with linter later 👍🏼

**Pull request merged**